packages/engine-core/src/binary/BinaryEngine.ts

```diff
@@ -188,7 +186,7 @@ export class BinaryEngine extends Engine<undefined> {
 
     if (removedFlagsUsed.length > 0 && !process.env.PRISMA_HIDE_PREVIEW_FLAG_WARNINGS) {
       console.log(
-        `${chalk.blueBright('info')} The preview flags \`${removedFlagsUsed.join(
+        `${blue(bold('info'))} The preview flags \`${removedFlagsUsed.join(
```

Comment:
these log level labels should probably be consistently bold or normal font, I like the idea of making them all bold